tools/wptrunner/wptrunner/environment.py

```diff
@@ -232,22 +230,25 @@ def get_routes(self):
                  "text/javascript", "/_pdf_js/pdf.js"),
                 (os.path.join(here, "..", "..", "third_party", "pdf_js", "pdf.worker.js"), None,
```

Comment:
Is there a difference between `None` and `{}` in L231 and L233? Maybe we should stick with one form.